ENH: add NaiveRatioUniforms from UNU.RAN to scipy.stats (#14717)
scipy/scipy: doc/source/tutorial/stats/sampling_nrou.rst

.. _sampling-nrou:

Naive Ratio-Of-Uniforms (NROU) Method
=====================================

.. currentmodule:: scipy.stats

* Required: PDF
* Optional: mode, center, bounding rectangle for acceptance region
* Speed:

  * Set-up: fast (if the bounding rectangle is known), otherwise slow
  * Sampling: moderate

> NROU is an implementation of the (generalized) ratio-of-uniforms method which

Reviewer: ```suggestion
NROU is an special case of the (generalized) ratio-of-uniforms method that
```
Is that what is meant?

Author: no, NROU is the usual ROU method if r==1, otherwise it is called generalized ROU
